common/lib/xmodule/xmodule/js/src/video/09_save_state_plugin.js

```diff
@@ -91,8 +93,19 @@ function() {
             this.saveState(true, {youtube_is_available: youtubeIsAvailable});
         },
 
+        onSkip: function (event, doNotShowAgain) {
+            var info = {date_last_view_bumper: true};
+            if (doNotShowAgain) {
+                _.extend(info, {do_not_show_again_bumper: true});
+            }
+            this.saveState(true, info);
```

Comment:
I think, we should send this data when we resolve the videoBumper ('skip', 'ended', 'error' events).